common/logisticspipes/proxy/specialinventoryhandler/AEInterfaceInventoryHandler.java

```diff
@@ -76,84 +86,63 @@ public SpecialInventoryHandler getUtilForTile(TileEntity tile, EnumFacing dir, b
 	}
 
 	@Override
-	public Map<ItemIdentifier, Integer> getItemsAndCount() {
+	public HashMap<ItemIdentifier, Integer> getItemsAndCount() {
```

Comment:
Why change this from Map to HashMap? The implementation you get doesn't matter in most cases

Reply:
I didn't change it. I made a new class, extended the special inventory handler and let intelliJ auto generate my functions. When i was done i simply copy-pasted it over the old class.

Reply:
Weird, since `IInventoryUtil` where this method comes from, specifies `Map` as the return type and I thought IntelliJ just kept that. Other classes also have HashMap as the return type though, so leave it I guess

Reply:
I will change it back.

Reply:
Then also do the same in the method right below this one, please